### PR TITLE
Add licenses to build.edn

### PR DESCRIPTION
**Pull request by makinoshi**

### CODE REVIEW

File: build.edn [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {:lib toyokumo/toyokumo-commons
  :version "0.3.{{git/commit-count}}"
  :description "Utility functions that are used in multiple Toyokumo products"
+ :licenses [{:name "Apache License Version 2.0,"
```
Comment (liquidz) [inline build.edn]: NIT The name does not have to be exact, so it is NIT, but there is a comma at the end.

**Review by liquidz — APPROVED** [review]
@makinoshi Sorry I didn't point this out first 🙇

File: build.edn (hunk 1 above)
Comment (makinoshi) [reply build.edn]: Fixed 38b35a3df3ddf22de10701114450fecd836a7119 😃

**Review by liquidz — APPROVED** [review]
@makinoshi Thanks! LGTM

**Pull request merged**